src/planning.py

```diff
@@ -264,46 +288,52 @@ def _run_low_level_search(task: Task, option_model: _OptionModelBase,
         if option.initiable(state):
             try:
                 next_state = option_model.get_next_state(state, option)
```

Comment:
shuffled around some of the lines in this part to enhance readability (impossible to understand from diffs, but the actual code is clearer than before)